registry/import.go

```diff
@@ -7,13 +7,58 @@ import (
 	"fmt"
 	"github.com/cozy/cozy-apps-registry/asset"
 	"github.com/cozy/cozy-apps-registry/config"
-	"github.com/go-kivik/kivik"
 	"io"
-	"io/ioutil"
 	"path"
 	"strings"
 )
 
+type couchDb struct {
+	db        string
+	documents []interface{}
+}
+type couchDbs map[string]*couchDb
+
+func (db *couchDb) bulkImport() error {
+	docs := db.documents
+	fmt.Printf("Bulk import %d CouchDB documents into %s\n", len(docs), db.db)
```

Comment:
I would add `if len(docs) == 0 { return nil }`, as I think we can have this case with the final `flush` if the number of imported docs is a multiple of 100, and `c.BulkDocs` will probably return an error when called with 0 docs.